docs/files/ros2-action-client.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import print_function
+import roslibpy
+import time
+
+
+global result
+
+client = roslibpy.Ros(host='localhost', port=9090)
+client.run()
+
+action_client = roslibpy.ActionClient(client,
+                                      '/fibonacci',
+                                      'custom_action_interfaces/action/Fibonacci')
+result = None
+
+def result_callback(msg):
+    global result
+    result = msg['result']
+
+def feedback_callback(msg):
+    print('Action feedback:',msg['partial_sequence'])
+
+def fail_callback(msg):
+    print('Action failed:',msg)
```

Comment:
These 2 will not have a space between the `:` and the message, so minimally, please add that.

I will also share my preference for f-strings, but this is optional

```python
print(f"Action failed: {msg}")
```